projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/collections/NamedStructureOutlierSet.java

```diff
@@ -36,11 +42,11 @@
 
   @JsonCreator
   public NamedStructureOutlierSet(
-      String structType,
-      String name,
-      T namedStructure,
-      SortedSet<String> conformers,
-      SortedSet<String> outliers) {
+      @JsonProperty(STRUCT_TYPE_VAR) String structType,
+      @JsonProperty(NAME_VAR) String name,
+      @JsonProperty(NAMED_STRUCTURE_VAR) T namedStructure,
+      @JsonProperty(CONFORMERS_VAR) SortedSet<String> conformers,
```

Comment:
Also, in my understanding, this might also be null? Or they are checked some place else?